### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by rebekah-west** [vladiatrinh/orie4741proj-vt95-yie3-tt426#7]
Loved this project! Great job!

This project uses data on building damage from the Gorkha earthquake to predict the damage grade caused by earthquakes to buildings in Nepal. In the midterm report, they went through thorough data preprocessing to identify the most relevant features and scale them appropriately. They were also thoughtful about which models might perform the best since this is an ordinal classification problem. 

I love how specific you are about what is contained in the data, both the number of features and the types of data each feature is (ordinal, categorical, etc). I also really liked how specific you were about preprocessing your data, particularly in being aware of how to scale age or the impact a uniform scaling might have. The various plots that you did for each feature were helpful for understanding the relationship between a feature and the damage grade. I think these plots will be particularly helpful with interpreting your final model. You also did a great job including techniques from class, both in the preprocessing and when thinking about model complexity and overfitting.

For improvement, I think it would be interesting to see how well your model would generalize to buildings outside Nepal. I also like your idea of diving deeper into hyperparameter tuning in order to improve the accuracy. Finally, I think it could be beneficial to explain/interpret the model after reporting its accuracy so that readers can understand which of the features from preprocessing did indeed become important in the trained model. With this, it could be interesting to compare the analysis of the features from the Exploratory Data Analysis section with the coefficients of the features in the model and see where the biggest differences arose.
